src/main/kotlin/com/depromeet/piki/admin/extraction/AdminExtractionPolicyController.kt

```
@@ -37,17 +44,48 @@ class AdminExtractionPolicyController(
-            listView(model)
+            boardView(filter = null, model = model, selectedRoute = route.name)
+        }
+
+    @GetMapping("/{domain}")
+    fun detail(
+        @PathVariable domain: String,
+        model: Model,
+    ): String =
+        try {
+            detailView(adminExtractionPolicyService.find(domain), model)
+        } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)